doc/en/source/ngsildbasediotintegration.rst

```diff
@@ -2,14 +2,30 @@ Integrate FogFlow with NGSILD Based IoT agent
 ****************************************************
 
 This tutorial introduces how FogFlow could be utilized as an advanced data analytics framework to enable on-demand data analytics
-on top of the raw data captured in the NGSI-LD IoT agent. The following diagram shows a simple example of how to do this in details, mainly including four aspects with 10 steps
+on top of the raw data captured in the NGSI-LD IoT agent.
+ 
+**IOT AGENT:** An IoT Agent is a component that lets a group of devices send their data to and be managed from a Context Broker using their own native protocols. IoT Agents should also be able to deal with security aspects of the FIWARE platform (authentication and authorization of the channel) and provide other common services to the device programmer.The north port of the IoT Agent is one such component which can communicate using NGSI-LD and the IoT Agent then translates the result so that all interactions beneath this port occur using the native protocol of the attached devices.IoT Agents already exist or are in development for many IoT communication protocols and data models. Examples include the following: 
```

Comment:
An IoT Agent is a component that lets a group of devices send their data to and be managed from a Context Broker using their own native protocols. if someone want to know more about IoT Agent then can refer yxz_link.
There are several IoT Agent in the market which are in production or already workable.